docs/team/sp4ce-cowboy.md

```diff
@@ -3,156 +3,164 @@ layout: page
-device running Java 11 or above, and it can be run locally with no network access required.
-
-This project was developed in a team of 5 members, as part of the course requirements for 
-[CS2103 Software Engineering](https://nusmods.com/courses/CS2103/software-engineering), in Semester 1 of AY23/24. 
-Given below are my contributions to the project, including the features that I've implemented, the
-enhancements that I've implemented, code and documentation that I've contributed and the team tasks that
-I have been involved in.
-
----
-
-### 2 – Project Contributions
-
-#### Functional Features
-
-* **Delete Transaction Command**:
-  * What it does: A CLI command to remove a transaction from UniCa$h, as specified by the user.
-  * Justification: As a finances management app, removal of transactions is an important core utility
-    especially if the user had entered the information incorrectly or if the user wants to
-    delete a transaction in general.
-  * Credits: AddressBook3's implementation of the `Delete` command
-
-* **Clear Transactions Command**:
-  * What it does: A CLI command to remove all transactions from UniCa$h.
-  * Justification: A user might want to clear all their previous transactions and
-    start over on a blank slate.
-
-
-* **Reset UniCash**: *to be added*
-* **Help Command** *to be added*
-* **Get Transactions**: *to be added*
-
-
-* **Feature Enhancements**
-  * **Find Transactions**: Added enhancements *to be added*
-
-#### User Interface Features
+for users who are proficient with CLI. It is written in Java 11, spanning about [20 KLoC](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=T16-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22&isTabOnMergedGroup=true&tabOpen=true&tabType=authorship&tabAuthor=elhy1999&tabRepo=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=true&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false).
+Code contributed by me: [RepoSense](https://nus-cs2103-ay2324s1.github.io/tp-dashboard/?search=T16-3&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code&since=2023-09-22&tabOpen=true&tabType=authorship&tabAuthor=sp4ce-cowboy&tabRepo=AY2324S1-CS2103-T16-3%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false). A summary of my contributions are given below, with more detailed
```

Comment:
Add an additional line above this so it's formatted with newlines, right now it looks like:

![Screenshot 2023-11-12 at 10 28 03](https://github.com/AY2324S1-CS2103-T16-3/tp/assets/29497029/72210929-94ca-46c8-9c4e-611f199d1dc7)

Reply:
Actually that is the intention, for this comment and the one below. I had to remove line breaks whereever possible because of the 2 page constraint, else it would exceed the 2 page hard limit. I had some other formatting issues so I emailed prof with a draft, he said it was fine to do so as long as it was reasonably legible.